src/ContainerHealthLogs.jsx

```diff
@@ -42,17 +42,10 @@ const HealthcheckOnFailureActionText = {
     kill: _("Force stop"),
 };
 
-const ContainerHealthLogs = ({ container, containerDetail, onAddNotification, state }) => {
-    let healthCheck = {};
-    let failingStreak = 0;
-    let logs = [];
-    if (containerDetail) {
-        healthCheck = containerDetail.Config.Healthcheck || containerDetail.Config.Health;
-        healthCheck.HealthcheckOnFailureAction = containerDetail.Config.HealthcheckOnFailureAction;
-        const healthState = containerDetail.State.Healthcheck || containerDetail.State.Health;
-        failingStreak = healthState.FailingStreak || 0;
-        logs = [...(healthState.Log || [])].reverse();
-    }
+const ContainerHealthLogs = ({ container, onAddNotification, state }) => {
+    const healthCheck = container.Config?.Healthcheck ?? container.Config?.Health ?? {};
+    const healthState = container.State?.Healthcheck ?? container.State?.Health ?? {};
+    const logs = [...(healthState.Log || [])].reverse();
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1350-20230712-152457-eee84e1f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)